policy-controller/grpc/src/inbound.rs

```diff
@@ -327,3 +330,48 @@ fn to_authz(
         authentication: Some(authn),
     }
 }
+
+fn to_rate_limit(rl: &RateLimit) -> proto::HttpLocalRateLimit {
+    let meta = Metadata {
+        kind: Some(metadata::Kind::Resource(api::meta::Resource {
+            group: "policy.linkerd.io".to_string(),
+            kind: "httplocalratelimitpolicy".to_string(),
```

Comment:
```suggestion
            kind: "HTTPLocalRateLimitPolicy".to_string(),
```

Reply:
I was following the convention from the other Kinds in this file. Is it ok to deviate in this particular case given the length of this name?

Reply:
I think the conventions are mixed, but the most desirable outcome (from a readability and overall consistency point of view) is to use the canonical form with capitalization. Specifically we have simply pass-through the Kind for route resources:

https://github.com/linkerd/linkerd2/blob/8fb22d697d580c43dbed89a120b83246fb7d5437/policy-controller/grpc/src/inbound/grpc.rs#L56-L63
https://github.com/linkerd/linkerd2/blob/8fb22d697d580c43dbed89a120b83246fb7d5437/policy-controller/grpc/src/inbound/http.rs#L56-L61
https://github.com/linkerd/linkerd2/blob/8fb22d697d580c43dbed89a120b83246fb7d5437/policy-controller/k8s/index/src/inbound/index.rs#L1837